src/cfnlint/rules/conditions/Used.py

```diff
@@ -43,10 +43,6 @@ def match(self, cfn):
                     ref_conditions.append(iftree[-1][0])
                 else:
                     ref_conditions.append(iftree[-1])
-            # Get resource's Conditions
-            for _, resource_values in cfn.get_resources().items():
-                if 'Condition' in resource_values:
-                    ref_conditions.append(resource_values['Condition'])
```

Comment:
Isn’t this still necessary to see if a Condition is used?

Reply:
It was a duplicate. The Resource conditions are added twice 😉